schedule-tracking/src/main/java/org/motechproject/scheduletracking/domain/json/ScheduleWindowsRecord.java

```diff
@@ -5,14 +5,21 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * <code>ScheduleWindowsRecord</code> is used to deserialize milestone window from json format.
```

Comment:
it is not used to "deserialize"; it just keeps the deserialized form of a schedule